html-extract.js

```diff
@@ -41,7 +42,7 @@ module.exports = function (opts) {
           self.push(new gutil.File({
             // Name: id or tag + index.
             path: file.path + "-" + (el.attribs.id || el.tagName + "-" + i),
-            contents: new Buffer(el.children[0].data)
+            contents: new Buffer(trim.left(el.children[0].data))
```

Comment:
And one other random comment -- when I've been wrangling analogous trimming issues in other projects, the situation often comes up of

```js
  var foo = true;
  if (foo) {
    bar = "hi"
  }
```

And when you want to trim, you really want to "find the number of spaces on the first real line" (here 2) and then "trim that number's worth of spaces everywhere" to produce:

```js
var foo = true;
if (foo) {
  bar = "hi"
}
```

Is there a chance we're going to want that scenario instead?